src/panel/resultTable.tsx

```diff
@@ -24,11 +24,34 @@ interface ResultTableProps<G> {
     renderGroup: (group: G) => ReactNode;
 }
 @observer export class ResultTable<G> extends PureComponent<ResultTableProps<G>> {
+    // Helper function to create span with conditional tooltip
+    private createSpanWithConditionalTooltip = (text: string, children?: React.ReactNode) => {
+        const content = children ?? text;
+        return (
+            <span
+                ref={(element) => {
+                    if (element && text) {
+                        // Check for truncation after layout
+                        setTimeout(() => {
```

Comment:
All these `setTimeout`s seem incorrect, but I'm not a TypeScript developer so I don't really know...